config.m4

```diff
@@ -1,4 +1,6 @@
-dnl config.m4 for extension mongodb
+dnl Determine the base directory used for subsequent m4_include calls
+m4_define(PHP_MONGODB_BASEDIR, esyscmd(printf %s "$(dirname "__file__")"))
```

Comment:
This was adapted from https://github.com/DataDog/dd-trace-php/commit/e279e5dd8b6abbe58df6cb4bdd10171a6c5d7e0d